troposphere/static/js/actions/ImageVersionActions.js

```diff
@@ -23,8 +23,11 @@ export default {
             let err_message = error_json;
             if (error_json.hasOwnProperty("non_field_errors")) {
                 err_message = error_json.non_field_errors.join(" , ");
+                if(err_message.name  != null && err_message.name[0] != null) {
+                    err_message = err_message.name[0]
+                }
             }
-            var message = "Error updating Image Version " + version.get("name") + ": " + err_message;
+            var message = "Error updating Image Version '" + version.get("name") + "': " + err_message;
```

Comment:
The reason i put `''` around it is because the version that failed was in fact an empty string.. in that case, the error message would read:

`Error updating Image Version : [object Object]` (originally -- now will show the error message string which would look something like this: `Error updating Image Version : Name not provided`.

Probably the best solution here would be to fix it so that it uses the original name value, or say something about "Updating to name X"..?